docs/libcurl/libcurl-multi.3

```diff
@@ -98,7 +98,12 @@ period for your select() calls.
 one of its input arguments, and by reading that you can figure out when all
 the transfers in the multi handles are done. 'done' does not mean
 successful. One or more of the transfers may have failed. Tracking when this
-number changes, you know when one or more transfers are done.
+number changes, you know when one or more transfers are done. Please note, that
+transfer may fail immediately, especially in case you are adding/re-using easy
```

Comment:
1. "a transfer" or "transfers" (not "transfer")
2. Why is this likely to happen "especially" in the case you reuse handles? In my view, transfers are always roughly equally likely to fail.

Reply:
1. Fixed
2. Well, you are right, fixed.